### DISCUSSION TITLE
Allow set command height and status bar height to 0?

### DISCUSSION THREAD

**Discussion by tmpm697** [helix-editor/helix#8204] [Ideas]
`(n)vim` has these options:
```
cmdheight = 0
laststatus = 0
```
to disable command line and statusline.

**Comment by woojiq** [comment]
What is the purpose of such commands? I can't imagine a situation where I would need it. Afaik in vim you can't do `cmdheight=0` because it should be positive.

**Comment by tmpm697** [comment]
> What is the purpose of such commands? I can't imagine a situation where I would need it. Afaik in vim you can't do `cmdheight=0` because it should be positive.

clean UX/UI I think, basically I know eactly everything about current buffer and its state, having two lines reserved for `cmd` and `status` is redundant.

Yes I think `cmdheight=0` does not work with `(n)vim`, but you can use `noice.nvim` that leverage `cmd`

**Comment by pascalkuthe** [comment]
This seems super niche to me. We try to keep the config minimal and helix is somewhat more opinionated instead of a "do whatever you want" sandbox like vim (and even they don't have a config option for this). I am not a fan of adding a ton of different config options like this as it makes the codebase easier to maintain for very questionable utility (how much difference do two more lines of vertical space really make).

**Comment by radsoc** [comment]
This is a requested feature: https://github.com/helix-editor/helix/issues/1662
